<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>357</width>
    <height>141</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>321</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>Warning</string>
   </property>
   <widget class="QPushButton" name="AcceptCreditButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>70</y>
      <width>141</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Ok</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>You are not administrator!!!</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>AcceptCreditButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>AcceptCreditButton</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>130</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>212</x>
     <y>59</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
